src/tools/compiletest/src/lib.rs

```diff
@@ -300,6 +300,10 @@ pub fn parse_config(args: Vec<String>) -> Config {
         None => panic!("`--stage` is required"),
     };
 
+    let src_root = opt_path(matches, "src-root");
+    let src_test_suite_root = opt_path(matches, "src-test-suite-root");
+    assert!(src_test_suite_root.starts_with(&src_root));
```

Comment:
nit: Maybe add a assert message to this to make it immediately clearer from the error where the problematic paths are coming from

Reply:
Sure

Reply:
Added assert message to include the reason and both paths.